### PR TITLE
feat:added product page and filter

### PR DESCRIPTION
**Pull request by 05him**
added the product listing page with the filter, in the e commerce app

live URL:- https://05him.github.io/The_Dark_Store/

live URL for product page:-https://05him.github.io/The_Dark_Store/src/product-page/product-page.html

### CODE REVIEW

**Review by sunilvr — APPROVED** [review]
PR is good to be merged

File: src/product-page/product-page.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  font-size: 1em;
+}
+
+.card-price-final {
+  font-size: 1.3em;
+}
+
+.card-desc {
+  font-size: 0.8em;
+}
```
Comment (Sahil25061999) [inline src/product-page/product-page.css:66]: use rem if possible, if you have specific reason for using em than its okay

File: src/product-page/product-page.html [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+                        <span class="card-desc block">Hey This Is A Tensiosn Free Jacket While Wearing This You Will Go All Tension Free, So Buy Now!</span>
+                        <div class="card-btn-box flex-center">
+                            <button type="button" class="btn btn-primary block">Add To Cart</button>
+                            <button type="button" class="btn btn-secondary-outline block">Buy Now</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="card-box-vertical">
+                    <div class="card-img flex-center">
+                        <img class="card-badge-top-right" src="../../assets/red-heart.svg">
```
Comment (Sahil25061999) [inline src/product-page/product-page.html:158]: use button for wishlist icon

**Review by Sahil25061999 — APPROVED** [review]
Most part looks good, just a suggestion for homepage to add some kind of hover effect to navbar element rather than just a cursor.
For the products page, in the filter section, all the radio buttons are getting selected, so you need to work on that.
Can be merged after the changes.

File: src/product-page/product-page.css (hunk 1 above)
Comment (05him) [reply src/product-page/product-page.css:66]: used em here to adjust the font according to the size of the card

**Pull request merged**